tests/python/unittest/test_arith_rewrite_simplify.py

```diff
@@ -275,6 +275,7 @@ def test_add_index_simplify():
 def test_sub_index_simplify():
     ck = RewriteChecker()
     x, y, z = te.var("x"), te.var("y"), te.var("z")
+    a, b, c = tvm.tir.Any(), tvm.tir.Any(), tvm.tir.Any()
```

Comment:
```suggestion
    a, b = tvm.tir.Any(), tvm.tir.Any(), tvm.tir.Any()
```
`c` is no longer needed now

Reply:
Nice catch!